numba/targets/arrayobj.py

```diff
@@ -1839,6 +1839,30 @@ def np_unique_impl(a):
     return np_unique_impl
 
 
+@overload(np.repeat)
+def np_repeat(a, repeats):
+    if not isinstance(a, (types.Array,
+                          types.Sequence,
+                          types.BaseTuple,
+                          types.Number,
+                          types.Boolean,
+                          )
+                      ):
+        return
+    if not isinstance(repeats, types.Integer):
```

Comment:
For `axis=None` (default) `repeats` can be an array-like of integers which is then broadcast.

Reply:
arrays if integers with the correct size accepted as `repeats` arguments.